src/modules/repository/repository-module-manager.js

```diff
@@ -217,15 +218,15 @@ class RepositoryModuleManager extends BaseModuleManager {
         value2 = null,
         value3 = null,
     ) {
-        return this.getRepository('event').createEventRecord(
+        return influx.sendEventTelemetry(
```

Comment:
This should be done like this, I think that this function createEventRecord should be moved to telemetry module and than called from there